Workspace/src/xbox_controller/launch/state_select.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
+	<node name="teleop_node" pkg="xbox_controller" type="xbox" />
+	<node name="drive_lis_pub" pkg="xbox_controller" type="state" />
+	<node name="joy_launcher" pkg="joy" type="joy_node" args="usb-©Microsoft_Corporation_Controller_15D5CD4-joystick" />
```

Comment:
does the argument to select the xbox controller not require
`<param name="dev" type="string" value="/dev/input/by-id/usb-©Microsoft_Corporation_Controller_15D5CD4-joystick" />`
Look at http://wiki.ros.org/joy/Tutorials/WritingTeleopNode#Writing_a_Launch_File_to_Start_all_of_the_Nodes